frontend/src/utils/generalUtils.ts

```diff
@@ -133,14 +135,43 @@ export const validateSubmitTxInputs = (
   }
 }
 
+export const validateSubmitTxInputsSkyline = (
+  settings: ISettingsState, sourceChain: ChainEnum, destinationChain: ChainEnum,
+  destinationAddr: string, amount: string, bridgeTxFee: string, isNativeToken: boolean
+): string | undefined => {
+  const chain = isNativeToken ? destinationChain : sourceChain;
+  if (BigInt(amount) < BigInt(settings.minUtxoChainValue[chain])) {
+    return `Amount less than minimum: ${convertUtxoDfmToApex(BigInt(settings.minUtxoChainValue[chain]).toString(10))} ${fromChainToChainCurrency(chain)}`;
```

Comment:
instead of fromChainToChainCurrency(chain)
it should be:
isNativeToken ? fromChainToChainNativeToken(sourceChain) : fromChainToChainCurrency(sourceChain)